inc/html.class.php

```diff
@@ -3542,6 +3542,7 @@ static function initEditorSystem($name, $rand = '', $display = true, $readonly =
             remove_script_host: false,
             entity_encoding: 'raw',
             paste_data_images: $('.fileupload').length,
+            paste_word_valid_elements: 'b,strong,i,em,h1,h2',
```

Comment:
Huh? All those elements to copy paste an image?

Reply:
(or maybe I did not understand how this shoul work... :D)

Reply:
I just tried with example from tinymce documentation https://www.tiny.cloud/docs/plugins/paste/#paste_word_valid_elements